README.md

```diff
@@ -40,3 +40,37 @@ These packages will be produced as part of the standard build task.
  ```
   -$ ./gradlew build
  ```
+
+## Adding Root CA Certificates
+Self signing certificates can be very useful for local testing & development with SSL enabled. 
+If openEQUELLA is configured to use SSL with such a certificate, then the admin-console-package will fail to log in to the institution because of this error: 
```

Comment:
Does the admin console show this in the GUI (say, an error dialog), or in the terminal?

Reply:
In the terminal - I've been more specific now.